src/site/content/en/blog/appcache-removal/index.md

```diff
@@ -0,0 +1,211 @@
+
+### Origin trial deployment timing and logistics
+
+While the "reverse" origin trial officially starts with Chrome 84, you can [sign up](https://developers.chrome.com/origintrials/#/register_trial/1776670052997660673) for the origin trial today and add the tokens to your HTML and AppCache manifests. As your web app's audience gradually upgrades to Chrome 84, any tokens that you've already added will go into effect.
+
+Once you've added a token to your AppCache manifest, you can visit `chrome://appcache-internals` to confirm that your local instance of Chrome (version 84 or later) has properly associated the origin trial token with your manifest's cached entries. If your origin trial is recognized, you should see a field with `Token Expires: Tue Apr 06 2021...` on that page, associated with your manifest:
+
+<figure class="w-figure">
+  <img src="appcache-token.jpg" alt="chrome://appcache-internals interface showing a recognized token." width="550">
+</figure>
+
+## Testing prior to removal
+
+We strongly encourage you to migrate off of AppCache as soon as is feasible. If you want to test of removing AppCache on your web apps, use the `chrome://flags/#app-cache` [flag](https://www.chromium.org/developers/how-tos/run-chromium-with-flags) to simulate its removal. This flag is available starting with Chrome 84.
+
+## Migration strategies
+
+Service workers, which are widely supported in current browsers, offer an alternative to the offline experience provided by AppCache.
+
+We've provided a [polyfill library](https://github.com/GoogleChromeLabs/sw-appcache-behavior) that uses a service worker to replicate some of the functionality of AppCache, though it does not replicate the entire AppCache interface. In particular, it does not provide a replacement for the `window.applicationCache` interface or the related AppCache events.
+
+For more complex cases, libraries like [Workbox](https://developers.google.com/web/tools/workbox/) provide an easy way to create a modern service worker for your web app.
+
+### Service workers and AppCache are mutually exclusive
+
+While working on your migration strategy, please keep in mind that Chrome will disable AppCache functionality on any page that's loaded under the [control](https://developers.google.com/web/fundamentals/primers/service-workers/lifecycle#scope_and_control) of a service worker. In other words, as soon as you deploy a service worker that controls a given page, you're no longer able to use AppCache on that page.
+
+Because of this, we recommend that you do not attempt to migrate to service workers piece-by-piece. It would be a mistake to deploy a service worker that only contains some of your caching logic. You can not fall back on AppCache to "fill in the gaps."
```

Comment:
```suggestion
Because of this, we recommend that you do not attempt to migrate to service workers piece-by-piece. It would be a mistake to deploy a service worker that only contains some of your caching logic. You cannot fall back on AppCache to "fill in the gaps."
```